### DISCUSSION TITLE
g-helper (debloat.bat)

### DISCUSSION THREAD

**Discussion by Alckvmne** [seerge/g-helper#2650] [Q&A]
i couldnt seem to find the " debloat.bat " on this forum. needed help on this one
thanks

**Comment by seerge** [comment]
@Alckvmne hello. It’s now part of the app and is called “Stop” / “Start” Services button under Extra settings

**Reply by Alckvmne** [reply]
Can I see an example on how to do it?

**Reply by seerge** [reply]
Do what ? Press ``Stop`` button under ``Extra`` ?

**Reply by Alckvmne** [reply]
Yes
